File: fizzbuzz-java/src/main/java/com/github/ambersariya/FizzBuzz.java
package com.github.ambersariya;

public class FizzBuzz {
    public String convert(int number) {
        boolean multipleOfThree = number % 3 == 0;
        boolean multipleOfFive = number % 5 == 0;

        if (multipleOfFive && multipleOfThree){
            return "FizzBuzz";
        }

        if (multipleOfThree) {
            return "Fizz";
        }
        if (multipleOfFive) {
            return "Buzz";
        }

        return String.valueOf(number);
    }
}

File: fizzbuzz/src/fizzbuzz.ts
export function fizzbuzz(number: Number) {
  const isDivisibleByThree: boolean = Number(number) % 3 == 0;
  const isDivisibleByFive: boolean = Number(number) % 5 == 0;
  if (isDivisibleByThree && isDivisibleByFive) return "FizzBuzz";
  if (isDivisibleByThree) return "Fizz";
  if (isDivisibleByFive) return "Buzz";
  return number.toString();
}
